builtin/logical/ssh/backend_test.go

```diff
@@ -105,6 +105,13 @@ SeKWrUkryx46LVf6NMhkyYmRqCEjBwfOozzezi5WbiJy6nn54GQt
 `
 )
 
+type CertValidityValidationMethod int
```

Comment:
This type is used to branch the validation logic in the `validateSSHCertificate` function.